frontera.md

```diff
@@ -12,9 +12,17 @@ Then execute:
 
 ```
 module load tacc-apptainer
-singularity pull -F docker://seissol/training:latest
-singularity build -f my-training.sif singularity.def
-singularity run my-training.sif
+apptainer pull -F docker://seissol/training:latest
+apptainer build -f my-training.sif singularity.def
+apptainer run my-training.sif
+```
+
+You can also use the automatically generated container after pulling the docker container 
+
+```
+module load tacc-apptainer
+apptainer pull -F docker://seissol/training:latest
+apptainer run training_latest.sif
```

Comment:
What about adding
`ln -s ./training_latest.sif ~/my-training.sif`
for matching calls later:
`mpirun apptainer run ~/my-training.sif gmsh -3 tpv13_training.geo`

Reply:
@Aangniu : did you try
`ln -s ./training_latest.sif ~/my-training.sif`
for me when I do that, I get a broken link, that why I added the realpath.

Reply:
we have to update the label to singularity pull -F docker://seissol/training:hps-2024-frontera